frontends/beta-frontend/contexts/PlayerContext.tsx

```diff
@@ -0,0 +1,33 @@
+import { PlayerStats } from '@/data/SamplePlayer';
```

Comment:
I assume this file can be deleted now

Reply:
It makes sense for me to keep it in case I ever want to test something without fetching data from an API. I moved sampleColumn to the Grid component and renamed it because it wasn't really a sample column, they were the columns for the grid component. I just initially called it sampleColumn and never changed the name. 

If it's bad practice to keep sample data let me know and I'll remove SamplePlayer.

Reply:
I was referencing to `PlayerContext` file not sample data :P 
Keeping sample data makes sense